java/debugger.jpda.ui/test/unit/src/org/netbeans/modules/debugger/jpda/ui/breakpoints/LineBreakpointPanelTest.java

```diff
@@ -0,0 +1,63 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.netbeans.modules.debugger.jpda.ui.breakpoints;
+
+import org.junit.Test;
+import static org.junit.Assert.*;
+
+public class LineBreakpointPanelTest {
+
+    @Test
+    public void testClassNameIsValid() {
+        boolean result = LineBreakpointPanel.isClassNamePatternValid("java.lang.Integer");
```

Comment:
Wouldn't using assertTrue(LineBreakPointPanel......     be more readable?

Also staticly importing all methods from LineBreakPointPanel would make lines shorter, like ..

assertTrue(isClassNamePatternValid("java.lang.Integer"));

It's kind of obvoius  which class we are testing here.